src/st/lib_psp/st_init_lib.c

```diff
@@ -152,19 +152,19 @@ void InitEntityIds(void) {
 }
 
 void OVL_EXPORT(Load)(void) {
-    OVL_EXPORT(cutscene_data) = GetLangAt(
+    OVL_EXPORT(cutscene_data_offset_eight) = GetLangAt(
         8, (u8*)cutscene_data_en, (u8*)cutscene_data_fr, (u8*)cutscene_data_sp,
         (u8*)cutscene_data_ge, (u8*)cutscene_data_it);
 
-    D_psp_092A54F0 = GetLangAt(
+    OVL_EXPORT(cutscene_data) = GetLangAt(
         0, (u8*)cutscene_data_en, (u8*)cutscene_data_fr, (u8*)cutscene_data_sp,
         (u8*)cutscene_data_ge, (u8*)cutscene_data_it);
 
-    D_psp_092A54E8 = GetLangAt(
+    OVL_EXPORT(cutscene_data_copy) = GetLangAt(
         0, (u8*)cutscene_data_en, (u8*)cutscene_data_fr, (u8*)cutscene_data_sp,
         (u8*)cutscene_data_ge, (u8*)cutscene_data_it);
 
-    D_psp_092A54E0 = GetLangAt(
+    OVL_EXPORT(cutscene_data_offset_four) = GetLangAt(
```

Comment:
It looks like these BSS pointers were used to set the 0th, 4th, 8th bit of the cutscene script.
I don't really have a good idea for how to name these but open to suggestions.